utilities/mtv_migration.py

```
@@ -361,6 +392,27 @@ def wait_for_migration_complate(plan: Plan) -> None:
+                and fixture_store is not None
+            ):
+                try:
+                    migration_uid: str = _get_migration_uid(plan=plan)
+                    capture_populate_pod_logs(
+                        ocp_admin_client=ocp_admin_client,
+                        namespace=target_namespace,
+                        migration_uid=migration_uid,
+                        fixture_store=fixture_store,
+                    )
+                    logs_captured = True
+                except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)